components/explore/HotelRooms.tsx

```diff
@@ -32,6 +32,11 @@ const HotelRooms = ({
   const [filteredRooms, setFilteredRooms] = useState<RoomDetail[]>(roomDetails);
 
```

Comment:
Changes to HotelRoom.tsx is not required for the attached Dev tasks. Please revert the unnecessary code changes or attach the relevant dev tasks to this PR.